README.md

```diff
@@ -13,12 +13,42 @@ net inst eda, from(http://wbuchanan.github.io/eda)
 ## Use case
 Since Exploratory Data Analysis can take a substantial amount of time in addition to the time needed to clean/prep data, this is intended to be used as a program that would be called at the end of the workday/overnight to produce permutations of univariate and bivariate visualizations and tables.  Then instead of spending time coding myriad possible combinations of variables to examine, a researcher could browse through a PDF generated through LaTeX while the computer does the work of compiling the results for them.
 
+## Usage
+
+You need a dataset open in memory to use `eda`. There are two required options, `output` for the naming of the output files 
```

Comment:
There is also `using` syntax available as well.  If a user were to issue:

```Stata
eda using auto.dta, o("eda-report") root("./") 
```
It should load the `auto.dta` data set in the current working directory and execute the remainder of the program.

Reply:
That said, I definitely appreciate the help putting together some more informative documentation.

Reply:
Sorry I didn't get to this in time!

Reply:
No worries.  I ended up finding another small bug a week or so later that I patched locally but still need to push.